doc/_templates/index.html

```diff
@@ -119,6 +119,50 @@ <h4 class="feature-title">Interactive Data Exploration</h4>
     </div>
   </div>
 
+  <!-- Feature Section 4 -->
+  <div class="row mb-0 flex-md-row-reverse  row-padding-between-features">
+    <div class="col-md-5">
+      <h4 class="feature-title">Tune arbitrary data wrangling</h4>
+      <h5 class="feature-title">Inspect it, apply it to new data</h5>
+      <p class="feature-text">Chain an arbitrary set of operations to prepare,
+      transform, assemble <strong>multiple tables</strong> for machine
+      learning, and then <strong>tune the full pipeline,
+      inspect it</strong>, or apply it to new data.
+      </p>
+      <p class="feature-text">
+	Works with any computational or dataframe engine.
+      </p>
+      <div class="pipeline-img">
+	{% include "../generated_for_index/pipeline.svg" %}
+      </div>
+      <p class="feature-text-see-also">
+        <a href="auto_examples/expressions/10_expressions_intro.html">Discover the skrub expressions →</a>
+      </p>
+    </div>
+    <div class="col-md-7">
+        <details>
+          <summary>Given two input dataframes
+            <code class="docutils literal notranslate"><span
+class="pre">products_df</span></code> and <code class="docutils literal notranslate"><span
+class="pre">baskets_df</span></code>: <i>(expand for full code)</i></summary>
+	    {% include "../generated_for_index/code_block_0.html" %}
+	</details>
+	<div class="feature-flex">
```

Comment:
```suggestion
	<div class="feature-flex" style="column-gap: 0.25em;">
```

my last suggestion would be to add a hairline gap between the 2 dataframes otherwise they look like a single dataframe when they touch